kongsberg:
  kmall_connection:
    type: "UDP"
    udp_address: "127.0.0.1"
    udp_tx: 60001
    udp_rx: 60000
    buffer_size: 35000
  kctrl_connection:
    type: "UDP"
    udp_rx: 4002
    udp_address: "127.0.0.1"
    udp_tx: 14002
  sounder_name: EM2040_130
  # custom kongsberg services
  ping_service: ping_cmd
  power_service: power_cmd
  settings_service: settings_cmd
  values_service: values_cmd
  # custom mb msgs
  mbraw_topic: mbraw
  offset_topic: pu_time_offset
  pointcloud_topic: pointcloud
  wc_topic: wc
  run_startup: false